Lib9c/Action/JoinArena.cs

```diff
@@ -67,6 +67,21 @@ public override IAccountStateDelta Execute(IActionContext context)
                     $"[{nameof(JoinArena)}] Aborted as the avatar state of the signer failed to load.");
             }
 
+            if (!avatarState.worldInformation.TryGetUnlockedWorldByStageClearedBlockIndex(
+                    out var world))
+            {
+                throw new NotEnoughClearedStageLevelException(
+                    $"{addressesHex}Aborted as NotEnoughClearedStageLevelException");
+            }
+
+            if (world.StageClearedId < GameConfig.RequireClearedStageLevel.ActionsInRankingBoard)
```

Comment:
just curiously, is there any reason not using `worldInformation.IsStageCleared`?